.github/workflows/GHPages.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         uses: actions/checkout@v4
       - uses: actions/setup-node@v4
       - name: Install Packages
-        run: npm i
+        run: npm i -f
```

Comment:
nits: you can add `force=true` to the .npmrc, to avoid repeating. :)

Reply:
Thank you! I didn't know that. That's useful.